### ISSUE TITLE
Patch 4.4 update?

### ISSUE THREAD

**Issue by LostLogia4** [doxxx/ffxiv-craft-opt-web#247]
I've waited for almost a week and the website still don't have the 4.4 synthesis items.

https://ermad.github.io/ffxiv-craft-opt-web/app/#/simulator

**Comment by doxxx** [comment]
That is not the official website, it is the testing website of a frequent contributor. The official website is http://ffxiv.lokyst.net or http://ffxiv-beta.lokyst.net for the latest updates.

I don't follow FFXIV news very closely so I was unaware of the new patch. Thanks for letting me know.

However, it appears that there are now 4-star recipes, for which my importer script does not know how to calculate their effective level (level 70 3-star recipes are effectively level 350, for example). The effective level progression of level 70 star recipes so far has been: 290, 300, 320, 350. I could infer that 4-star recipes are 390 but it would just be a guess. I need confirmation from somebody who can test it.

@ermad Have you been able to determine this yet?

**Comment by doxxx** [comment]
Given that level 60 4-star recipes were effectively 40 levels higher than level 60 3-star recipes, that seems to be a safe bet. So I can provisionally import the database using that progression and if it's later determined to be wrong, I can reimport the database.

**Comment by Ermad** [comment]
I am not playing atm either (taking a break for a bit), so why I haven't done any updates yet. Most of the stuff follow the same formula so should be easy to update regardless.

If anyone has some data from the 4* crafts I can use to confirm the results that will help a lot.

**Comment by doxxx** [comment]
Deployed to beta site.

**Comment by yls888** [comment]
Ing2RecipeLevelTable
380: 293   // 70_4star

Ing1RecipeLevelTable
380: 294   // 70_4star

ProgressPenaltyTable
380: -0.048

QualityPenaltyTable
380: -0.1305

I have measured these data and there will be some errors in the advancement of the high progress, but these errors are within acceptable limits.

I do not guarantee its correctness and it can be only used as a reference.

Interestingly, when testing this data, I found that the parameters of the control calculation were very close to a multiple of the number 0.345, but I was not sure exactly where it took the integer.

`baseQuality = 3.4500e-5 * control * control + 0.34500 * control + 34.50

var QualityPenaltyTable = {
	380: -0.12
}`
